src/map/clif.c

```diff
@@ -10797,7 +10797,7 @@ static void clif_parse_LoadEndAck(int fd, struct map_session_data *sd)
 
 	// Spawn pet.
 	if (sd->pd != NULL) {
-		if (battle_config.pet_no_gvg != 0 && map_flag_gvg2(sd->bl.m)) { // Return the pet to egg. [Skotlex]
+		if (map->list[sd->bl.m].flag.nopet == 1) { //Return the pet to egg. [Skotlex]
```

Comment:
```suggestion
		if (map->list[sd->bl.m].flag.nopet != 0) { //Return the pet to egg. [Skotlex]
```
Just to be consistent...

Reply:
updated